src/Symfony/Component/Console/Tests/Style/SymfonyStyleTest.php

```diff
@@ -57,7 +57,7 @@ public function inputCommandToOutputFilesProvider()
 
     public function testLongWordsBlockWrapping()
     {
-        $word = 'Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon';
```

Comment:
As we discussed, this change should not have been necessary for the test to pass. So I suggest to see #18896